dumb-jump.el

```diff
@@ -943,8 +927,9 @@ denoter file/dir is found or uses dumb-jump-default-profile"
 
 (defun dumb-jump-goto-file-line (thefile theline pos)
   "Open THEFILE and go line THELINE"
-  ;(dumb-jump-message "Jumping to file '%s' line %s" thefile theline)
-  (ring-insert find-tag-marker-ring (point-marker))
+  (if (fboundp 'xref-push-marker-stack)
+      (xref-push-marker-stack)
```

Comment:
I just got this package installed and think, "hmm, `dumb-jump-back`, this could use `xref` pop..." I just try it for fun and it works!  I forever love you @purcell <3

Reply:
Emacs conventions are a wonderful thing!